src/package_io/parse_input.star

```diff
@@ -94,6 +94,9 @@ def parse_input(plan, input_args):
 
     result["tx_spammer_params"] = get_default_tx_spammer_params()
     result["goomy_blob_params"] = get_default_goomy_blob_params()
+    result["custom_flood"] = dict(
```

Comment:
can we call this `custom_flood_parms`; also can you follow the `tx_spammer_params` way of doing it; it will jsut work and its easier to read